src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/Strings.resx

```diff
@@ -132,30 +132,6 @@
   <data name="FileResolved" xml:space="preserve">
     <value>Input file '{0}' is resolved to new relative path '{1}' at directory '{2}'.</value>
   </data>
-  <data name="MarkupCompilePass1Task" xml:space="preserve">
```

Comment:
Was there a hint in the resx xml previously that indicated that this item ("MarkupCompilePass1Task") should not have been localized?

Reply:
This is no way to represent that data in the resx file.